xbmc/Application.h

```diff
@@ -483,26 +479,32 @@ class CApplication : public CXBApplicationEx, public IPlayerCallback, public IMs
   void HandleWinEvents();
 
   /*! \brief Helper method to determine how to handle TMSG_SHUTDOWN
-  */
+   */
   void HandleShutdownMessage();
 
-  CInertialScrollingHandler *m_pInertialScrollingHandler;
+  CInertialScrollingHandler* m_pInertialScrollingHandler;
 
   ReplayGainSettings m_replayGainSettings;
-  std::vector<IActionListener *> m_actionListeners;
-  std::vector<std::string> m_incompatibleAddons;  /*!< Result of addon migration */
+  std::vector<IActionListener*> m_actionListeners;
+  std::vector<std::string> m_incompatibleAddons; /*!< Result of addon migration */
   std::deque<XBMC_Event> m_winEvents;
 
 private:
-  CCriticalSection m_critSection;                 /*!< critical section for all changes to this class, except for changes to triggers */
-
-  CCriticalSection m_frameMoveGuard;              /*!< critical section for synchronizing GUI actions from inside and outside (python) */
-  std::atomic_uint m_WaitingExternalCalls;        /*!< counts threads wich are waiting to be processed in FrameMove */
-  unsigned int m_ProcessedExternalCalls;          /*!< counts calls wich are processed during one "door open" cycle in FrameMove */
-  unsigned int m_ProcessedExternalDecay = 0;      /*!< counts to close door after a few frames of no python activity */
+  CCriticalSection m_critSection; /*!< critical section for all changes to this class, except for
+                                     changes to triggers */
+
+  CCriticalSection
+      m_frameMoveGuard; /*!< critical section for synchronizing GUI actions from inside and outside
```

Comment:
This looks... weird. Unintended side effect?

Reply:
it's because of the comment causing the  line to be too long and getting wrapped and that's the only place to wrap it and keeping the comment as an end of line comment.

Not sure if there's a good solution to it actually but I agree that it looks weird.